modules/store/spec/store.spec.ts

```diff
@@ -587,6 +589,204 @@ describe('ngRx Store', () => {
         complete: done,
       });
     });
+
+    describe('Via tokens', () => {
+      const createReducer = ({
+        logText,
+        log,
+      }: {
+        logText: string;
+        log: string[];
+      }) => (reducer: ActionReducer<any, any>) => (
+        state: any,
+        action: Action
+      ) => {
+        log.push(logText);
+        return reducer(state, action);
+      };
+
+      it('should be called when the `metaReducers` option is not defined', () => {
+        const log: string[] = [];
+        TestBed.configureTestingModule({
+          imports: [StoreModule.forRoot({})],
+          providers: [
+            {
+              provide: META_REDUCERS,
+              useValue: createReducer({ logText: 'first', log }),
+              multi: true,
+            },
+            {
+              provide: META_REDUCERS,
+              useValue: createReducer({ logText: 'second', log }),
+              multi: true,
+            },
+          ],
+        });
+
+        store = TestBed.get(Store) as Store<any>;
+
+        expect(log).toEqual(['first', 'second']);
+        store.dispatch({ type: 'FOO' });
+        expect(log).toEqual(['first', 'second', 'first', 'second']);
+      });
+
+      it('should respect the order of reducers', () => {
+        const log: string[] = [];
+        TestBed.configureTestingModule({
+          imports: [
+            StoreModule.forRoot(
```

Comment:
This should include a test with an additional NgModule that contains a provided meta-reducer. NgModules have priority over ModuleWithProviders so it will change the order